goodmap/core_api.py

```diff
@@ -34,7 +35,8 @@ def get(self):
 
     @core_api.route("/languages")
     class Languages(Resource):
-        '''dupa'''
+        """dupa"""
+
```

Comment:
this for sure should not be there. I know that you didn't put it, but could you clean this up? :)